tools/release.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+import copy
+import hashlib
+import json
+import re
+import requests
+import os
+import shutil
+
+version = "0.1.0"
+
+psoc_ino_root_path = os.path.relpath(
+    os.getcwd()
+)
+build_dir_name = "pkg_build"
+pkg_assets_build_path = os.path.join(psoc_ino_root_path, build_dir_name)
+
+
+def strip_prefix_from_version(version):
+    return re.sub(r"[vV]", "", version)
+
+
+def mkdir_package_dir(version):
+    semver = strip_prefix_from_version(version)
+    pkg_name = "PSOC_IFX_" + semver
+    pkg_build_path = os.path.join(pkg_assets_build_path, pkg_name)
+    os.mkdir(pkg_build_path)
+
+    return pkg_name
+
+
+def build_package(pkg_name):
+    pkg_build_path = os.path.join(pkg_assets_build_path, pkg_name)
+
+    dirs_to_copy = ["cores", "tools", "mtb-libs", "examples"]
+
+    files_to_copy = [
+        "boards.txt",
+        "platform.txt",
+        "LICENSE.md",
+        "README.md",
+    ]
+
+    for dir in dirs_to_copy:
+        shutil.copytree(
+            os.path.join(psoc_ino_root_path, dir), os.path.join(pkg_build_path, dir)
+        )
+
+    for file in files_to_copy:
+        shutil.copyfile(
+            os.path.join(psoc_ino_root_path, file), os.path.join(pkg_build_path, file)
+        )
+
+
+def zip_package(pkg_name):
+    pkg_build_path = os.path.join(pkg_assets_build_path, pkg_name)
+    shutil.make_archive(pkg_build_path, "zip", pkg_assets_build_path, pkg_name)
+
+
+def get_package_size(pkg):
+    return os.path.getsize(pkg)
+
+
+def get_package_sha256(pkg):
+    with open(pkg, "rb") as f:
+        bytes = f.read()
+        hash = hashlib.sha256(bytes).hexdigest()
+
+    return hash
+
+
+def get_latest_package_index_json():
+    return requests.get(
+        "https://github.com/Infineon/arduino-core-psoc/releases/download/0.0.0-none/package_psoc_index.json"
```

Comment:
This should point to latest URL, and ideally for development purposes we can make it a parameter somehow.